samples/tables/predict-gcs-source-bq-dest.v1beta1.js

```diff
@@ -39,38 +36,45 @@ async function main(
   // const outputUri = '[BIGQUERY_PATH]' e.g., "bq://<project_id>",
   // `The destination Big Query URI for storing outputs`;
 
+  const automl = require('@google-cloud/automl');
+
+  // Create client for prediction service.
+  const automlClient = new automl.v1beta1.PredictionServiceClient();
+
   // Get the full path of the model.
-  const modelFullId = client.modelPath(projectId, computeRegion, modelId);
+  const modelFullId = automlClient.modelPath(projectId, computeRegion, modelId);
 
-  // Get the multiple Google Cloud Storage input URIs.
-  const inputUris = inputUri.split(',');
-  const inputConfig = {
-    gcsSource: {
-      inputUris: inputUris,
-    },
-  };
+  async function batchPredict() {
+    const inputConfig = {
+      gcsSource: {
+        inputUris: [inputUri],
+      },
+    };
 
-  // Get the Big Query output URIs.
-  const outputConfig = {
-    bigqueryDestination: {
-      outputUri: outputUri,
-    },
-  };
+    // Get the Big Query output URIs.
+    const outputConfig = {
+      bigqueryDestination: {
+        outputUri: outputUri,
+      },
+    };
 
-  // Get the latest state of long-running operation.
-  client
-    .batchPredict({
+    const responses = await automlClient.batchPredict({
       name: modelFullId,
       inputConfig: inputConfig,
       outputConfig: outputConfig,
-    })
-    .then(responses => {
-      const operation = responses[1];
-      console.log(`Operation name: ${operation.name}`);
-    })
-    .catch(err => {
-      console.error(err);
     });
+
+    // Get the latest state of long-running operation.
+    const operation = responses[1];
+    console.log(`Operation name: ${operation.name}`);
+  }
+
+  batchPredict();
+
   // [END automl_tables_predict_using_gcs_source_and_bq_dest]
 }
-main(...process.argv.slice(2)).catch(console.error());
+
+if (module === require.main) {
+  runSample().catch(console.error);
```

Comment:
We've updated the guidance for error handling in these samples at:
https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head

Specifically, we want to do something like:
```js
main(...process.argv.slice(2)).catch(err => {
  console.error(err.message);
  process.exitCode = 1;
});
process.on('unhandledRejection', (err) => {
  console.error(err.message);
  process.exitCode = 1;
});
```

This only changed friday of last week (apologies)